packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -18,9 +18,13 @@ const double _singleLineVerticalPadding = 14.0;
 
 // TODO(ianh): We should check if the given text and actions are going to fit on
 // one line or not, and if they are, use the single-line layout, and if not, use
-// the multiline layout. See link above.
+// the multiline layout, https://github.com/flutter/flutter/issues/32782
+// See https://material.io/components/snackbars#specs, 'Longer Action Text' does
+// not match spec.
 
-// TODO(ianh): Implement the Tablet version of snackbar if we're "on a tablet".
+
+// TODO(ianh): Implement the Tablet version of snackbar if we're "on a tablet",
```

Comment:
I think you can kill this TODO. we have an issue to track this as a feature request.